tests/foreman/api/test_contentmanagement.py

```diff
@@ -659,26 +627,26 @@ def test_positive_iso_library_sync(self, capsule_configured):
     @pytest.mark.tier4
     @pytest.mark.skip_if_not_set('capsule', 'clients', 'fake_manifest')
     def test_positive_on_demand_sync(self, capsule_configured, default_sat):
-        """Create a repository with 'on_demand' sync, add it to lifecycle
-        environment with a capsule, sync repository, examine existing packages
-        on capsule, download any package, examine packages once more
+        """Create a repository with 'on_demand' policy, add it to a CV,
+        promote to an 'on_demand' Capsule's LCE, check artifacts were created,
+        download a published package, assert it matches the source.
 
         :id: ba470269-a7ad-4181-bc7c-8e17a177ca20
 
         :expectedresults:
 
-            1. After initial syncing only symlinks are present on both
-               satellite and capsule, no real packages were fetched.
-            2. All the symlinks are pointing to non-existent files.
-            3. Attempt to download package is successful
-            4. Downloaded package checksum matches checksum of the source
-               package
+            1. A custom yum repository is successfully synced and ContentView published
```

Comment:
Thanks for cleaning up these statements and making them accurate for the updated test.